diff --git a/node_modules/react-native-gps-state/android/build.gradle b/node_modules/react-native-gps-state/android/build.gradle
index bc29af6..4fff137 100644
--- a/node_modules/react-native-gps-state/android/build.gradle
+++ b/node_modules/react-native-gps-state/android/build.gradle
@@ -46,8 +46,8 @@ repositories {
 }
 
 dependencies {
-    compile "com.facebook.react:react-native:+"  // From node_modules
-    compile 'com.android.support:appcompat-v7:26.1.0'
+    implementation "com.facebook.react:react-native:+"  // From node_modules
+    implementation 'com.android.support:appcompat-v7:26.1.0'
     /*androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })*/
